### PR TITLE
changed max pool leverage to 5

### PR DESCRIPTION
**Pull request by aaraujoxr**

### CODE REVIEW

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 1]
```diff
@@ -332,7 +332,7 @@ uniUsdcMaxStaleDurationFallbackStorkRedstone = "0"
 [var.market_1eth]
 market1Eth_riskMatrixIndex = "0"
 market1Eth_maxOpenBaseUnscaled = "37500"
-market1Eth_velocityMultiplierUnscaled = "80"
+market1Eth_velocityMultiplierUnscaled = "16"
```
[the remark this answers is not in the document]
Comment (arturbeg) [reply packages/tomls/src/omnibus/reya_cronos.toml:335]: or is this change independent of the leverage increase?

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 2]
```diff
@@ -550,7 +550,7 @@ cp1Rusd_mmrMultiplierUnscaled = "1"
 cp1Rusd_dutchMultiplierUnscaled = "1"
 cp1Rusd_adlMultiplierUnscaled = "0.6"
 cp1Rusd_imBufferMultiplierUnscaled = "8.06"
-cp1Rusd_poolImMultiplierUnscaled = "8.01"
+cp1Rusd_poolImMultiplierUnscaled = "5.2"
```
Comment (arturbeg) [inline packages/tomls/src/omnibus/reya_cronos.toml:553]: hmm, should it be 8.01 * 0.6 = 4.8?

Comment (aaraujoxr) [reply packages/tomls/src/omnibus/reya_cronos.toml:553]: this actually incorporates two changes: the standardization of leverages (meaning, 8.01 should have been adjusted for the max leverage for ETH and BTC to actually be 20x), and then the increase in leverage. To check if it's alright: 1/(IMR_mult x sqrt(matrix_entry))=5

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 3]
```diff
@@ -342,15 +342,15 @@ market1Eth_slippageParams_betaUnscaled = "0"
 market1Eth_minimumOrderBaseUnscaled = "0.010"
 market1Eth_baseSpacingUnscaled = "0.001"
 market1Eth_priceSpacingUnscaled = "0.001"
-market1Eth_depthFactorUnscaled = "6"
+market1Eth_depthFactorUnscaled = "3.6"
 market1Eth_maxExposureFactorUnscaled = "0.999"
 market1Eth_maxPSlippageUnscaled = "0.1"
-market1Eth_marketOrderMaxStaleDuration = "3"
+market1Eth_marketOrderMaxStaleDuration = "0"
```
[the remark this answers is not in the document]
Comment (aaraujoxr) [reply packages/tomls/src/omnibus/reya_cronos.toml:348]: this is not intentional, no. not sure how i did it

File: packages/tomls/src/omnibus/reya_cronos.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (aaraujoxr) [reply packages/tomls/src/omnibus/reya_cronos.toml:335]: please change to 48.

File: packages/tomls/src/omnibus/reya_cronos.toml (hunk 3 above)
[the remark this answers is not in the document]
Comment (arturbeg) [reply packages/tomls/src/omnibus/reya_cronos.toml]: https://github.com/Reya-Labs/reya-deployments/pull/102/commits/0dc2fcbb37a51c3572bf31172267400ea6da5b4f

File: packages/tomls/src/omnibus/reya_cronos.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (arturbeg) [reply packages/tomls/src/omnibus/reya_cronos.toml]: https://github.com/Reya-Labs/reya-deployments/pull/102/commits/c5b175b54d10d2baf8c0e3c8516be45269c82d62

File: packages/tomls/src/omnibus/reya_cronos.toml (hunk 2 above)
Comment (arturbeg) [reply packages/tomls/src/omnibus/reya_cronos.toml:553]: sg

**Review by arturbeg — APPROVED** [review]

**Pull request merged**